src/components/News/News.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import style from './News.module.css'
```

Comment:
You forgot a new line above style import.

Reply:
/fxd